static/app/components/feedback/feedbackItem/useFeedbackActions.ts

```diff
@@ -27,13 +32,52 @@ const mutationOptions = {
 
 export default function useFeedbackActions({feedbackItem}: Props) {
   const organization = useOrganization();
+  const projectId = feedbackItem.project?.id;
+  const api = useApi();
+  const navigate = useNavigate();
+  const {pathname, query} = useLocation();
+  const hasDelete =
+    organization.access.includes('event:admin') &&
+    organization.features.includes('issue-platform-deletion-ui');
 
   const {markAsRead, resolve} = useMutateFeedback({
     feedbackIds: [feedbackItem.id],
     organization,
     projectIds: feedbackItem.project ? [feedbackItem.project.id] : [],
   });
 
+  const onDelete = useCallback(
+    closeModal => {
+      addLoadingMessage(t('Updating feedback...'));
+
+      bulkDelete(
+        api,
+        {
+          orgId: organization.slug,
+          projectId: projectId,
+          itemIds: [feedbackItem.id],
+        },
+        {
+          complete: () => {
+            closeModal();
+            navigate(
+              normalizeUrl({
+                pathname: pathname,
+                query: {
+                  mailbox: query.mailbox,
+                  project: query.project,
+                  query: query.query,
+                  statsPeriod: query.statsPeriod,
+                },
+              })
```

Comment:
what's this `navigate` for?

Reply:
to reload the page after deleting

Reply:
Can we add a comment explaining this? Would the pathname include the deleted feedback's id?

Reply:
the pathname is just `feedback` in this case, I can hardcode that instead